A43-sdk-authorization.md

```diff
@@ -294,10 +294,10 @@ Valid user provided SDK authorization policy creates authorization engine(s).
 In the case of file watcher, we internally create thread(C-core)/ goroutine(Go)/
 scheduled service(Java) which will be used to read the policy file periodically,
 and update the authorization engines. During the first file read, if the policy
-is invalid or there are I/O errrors, we will return error back to application and
-SDK authorization won't be enabled. If the error occurs on a later reload, then
-that particular reload will be skipped and error will be logged, and we will
-continue to use the latest valid policy to make authorization decisions.
+is invalid or there are I/O errrors, we will return error back to application.
```

Comment:
I think this should explicitly say something like "we will return an error back to the application, which will generally result in the gRPC server not starting."  That way, we make it clear what the impact of this API is.

Reply:
Sure. Updated text. Thank you!